FF-RN-26Jan23 (#487)
harness/developer-hub: release-notes/feature-flags.md

Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
:::

## January 26, 2023

### What's new

This release does not include new features.

### Early access

This release does not include early access features.

### Fixed issues
#### Feature Flag SDKs

> - The React Client SDK has been updated to version 1.1.0. This includes the following changes:

Reviewer: ```suggestion
- The React Client SDK has been updated to version 1.1.0 and includes the following changes:
```

  - You can now listen for errors that are caused by network issues. For more information about this, go to [the SDK's readme file.](https://github.com/harness/ff-react-client-sdk/blob/main/README.md) (FFM-6578)

>   - You can now provide the SDK with a set of evaluations that it can serve instantly on initialization. For more information about this, go to [the SDK's readme file.](https://github.com/harness/ff-react-client-sdk/blob/main/README.md) (FFM-6578) 

Reviewer: ```suggestion
  - You can now provide the SDK with a set of evaluations that it can serve instantly upon initialization. For more information about this, go to [the SDK's readme file.](https://github.com/harness/ff-react-client-sdk/blob/main/README.md) (FFM-6578) 
```

> - The Javascript SDK has been updated to version 1.8.0. This includes the following changes:

Reviewer: ```suggestion
- The Javascript SDK has been updated to version 1.8.0 and includes the following change:
```

>   - You can now provide the SDK with a set of evaluations that it can serve instantly on initialization. For more information about this, go to [the SDK's readme file.](https://github.com/harness/ff-javascript-client-sdk/blob/main/README.md) (FFM-6489) 

Reviewer: ```suggestion
  - You can now provide the SDK with a set of evaluations that it can serve instantly upon initialization. For more information about this, go to [the SDK's readme file.](https://github.com/harness/ff-javascript-client-sdk/blob/main/README.md) (FFM-6489) 
```
